client/app/pages/queries/QuerySource.jsx

```diff
@@ -154,20 +159,14 @@ function QuerySource(props) {
         return;
       }
       if (isDirty || !isEmpty(selectedText)) {
-        executeAdhocQuery(selectedText);
+        executeQuery(null, () => {
```

Comment:
Need to improve the API here so we don't have to pass `null` as first argument if we're passing in custom executor.